main.go

```diff
@@ -93,14 +96,27 @@ func main() {
 	}
 
 	// Migrate the schema
+	err = migration.Migrate(db)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// Nothing to here except exit, since the migration is already performed.
+	if migrateDB {
+		os.Exit(0)
+	}
+
 	ts := models.NewGormTransactionSupport(db)
 	witRepo := models.NewWorkItemTypeRepository(ts)
 	wiRepo := models.NewWorkItemRepository(ts, witRepo)
 
-	if err := transaction.Do(ts, func() error {
-		return migration.Perform(context.Background(), ts.TX(), witRepo)
-	}); err != nil {
-		panic(err.Error())
+	// Make sure the database is populated with the correct types (e.g. system.bug etc.)
+	if configuration.GetPopulateCommonTypes() {
+		if err := transaction.Do(ts, func() error {
+			return migration.Populate(context.Background(), ts.TX(), witRepo)
```

Comment:
This is a bit of a chicken & egg issue. Currently, we don't support 'proper' WIT update, so we hack it on a DB level. When WIT Update is in place, this should technically be reduced to a call to 'witrepo.Update(WIT)'. Long term we need a strategy for updating WIT's in a version controlled manner so we can 'view' a WI back in time using the WIT it had at that time.